### PR TITLE
Add Dockerfile and Makefile to simplify set up and execution of tests

### PR DESCRIPTION
**Pull request by marotpam**
Hi @JavierCane, I thought it might be useful to include a Dockerfile and Makefile, so people don't need to install anything on their local environment and to automate the installation of all the dependencies (php7, composer and the required packages). Do you think it's worth it?

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -0,0 +1,8 @@
+help: ## Prints this help.
+	@grep -E '^[a-zA-Z_-]+:.*?## .*$$' $(MAKEFILE_LIST) | awk 'BEGIN {FS = ":.*?## "}; {printf "\033[36m%-20s\033[0m %s\n", $$1, $$2}'
+
+build: ## Builds the docker container and installs dependencies
+	docker build -t php7boilerplate .
```
Comment (thisiserico) [inline Makefile]: I'd suggest using another name for the image, something that reflects that you're using it for the purpose of this kata.

**Comment by JavierCane** [comment]
Hi @marotpam!

Firstly, thanks a lot for your contribution! I really appreciate it because it will probably be useful for the people 💃  😄 

I don't have a lot of knowledge about Docker, so just one question: What do you think about declaring the volume in the `Dockerfile`?

As @aeony said (also thanks for reviewing the contribution), if you could change the `php7boilerplate` name for something like `finderKata`, I will go ahead merging the PR :)

Thanks!

**Comment by marotpam** [comment]
Hello again @JavierCane! Thanks for replying so quickly! I've renamed the container to finder-kata (docker doesn't allow capital letters in tag names). As for the Volume thing you suggested, I'd prefer to keep it how it is, otherwise it will be way slower on hosts running OS-X

**Comment by JavierCane** [comment]
Perfect then @marotpam. PR merged :)

See you!

**Pull request merged**